init.lua

```diff
@@ -12,46 +12,26 @@
 ]]--
 
 
-
-local wp = minetest.get_worldpath() .. "/luscious"
-minetest.mkdir(wp)
-
 local mgp = minetest.get_mapgen_params()
 local chunksize = 16 * mgp.chunksize
 
-local function cmpy(p2, y)
-	if y > 0 then
-		local h1 = p2 % 16
-		local h2 = math.floor(p2 / 16) * 16
-
-		h1 = h1 - math.max(math.min(h1, math.floor(y / 16)), 0)
-		return math.max(1, h1 + h2)
-	else
-		return math.max(1, p2)
-	end
+local function chose_color(pos)
+    -- Calculate color from biome heat and humidity.
+    local biome_data = minetest.get_biome_data(pos)
+    local hum1 = biome_data.humidity
+    local temp1 = biome_data.heat
+    local vh = math.floor(math.min(math.max(math.floor(hum1), 0), 100) / 6.6)
+	local vt = math.floor(math.min(math.max(math.floor(temp1), 0), 100) / 6.6)
+	local c1 = (vt + (vh * 16))
```

Comment:
extra parenthesis pair can be removed.

I'd Prefer:
```
local c1 = vh * 16 + vt
```
since no parentheses needed